src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -186,7 +217,7 @@ export default {
 
 	data() {
 		return {
-			focusTrap: null,
+			$focusTrap: null,
```

Comment:
Not a fan of local variables prefixed with `$`, always feels like this is a variable injected by the app (like `$router`).

Reply:
=> `focusTrapInstance`?